docker/configs/generators/generators/dot_properties.py

```diff
@@ -0,0 +1,47 @@
+import os
+import logging
+from . import *
+from cloudformation import get_stack_outputs
+
+LOGGER_NAME = os.path.splitext(os.path.basename(__file__))[0]
+LOGGER = logging.getLogger(LOGGER_NAME)
+OUTPUT_DIR = '/configs/etc/gu'
```

Comment:
made the output directory a required command line argument - https://github.com/guardian/grid/commit/876cdd492f03ad2f570a779e8b17381b2e0d9083